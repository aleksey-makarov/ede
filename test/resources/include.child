{# FIXME: 'var' should be available here #}
Hi, {{ scope }}!
