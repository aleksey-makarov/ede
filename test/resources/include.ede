{ "global": "some_thing" }
---
{% include "include.child1" %}
{% include "include.child2" with binding = "123" %}
