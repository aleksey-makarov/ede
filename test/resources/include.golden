global: some_thing!
binding: 123!
